swagger2.0/oic.r.rule.swagger.json

```diff
@@ -0,0 +1,371 @@
+                  ]
+                },
+                {
+                  "href": "/ruleactioncollection",
+                  "rt":   ["oic.r.rule.actioncollection"],
+                  "if":   ["oic.if.ll","oic.if.baseline"],
+                  "p":    {"bm": 3},
+                  "eps": [
+                    {"ep": "coap://[fe80::b1d6]:1111"}
+                  ]
+                }
+              ]
+            },
+            "schema": { "$ref": "#/definitions/sbaseline" }
+          }
+        }
+      }
+    },
+    "/RuleResURI?if=oic.if.create": {
+      "get": {
+        "description": "Retrieval response for an instance of oic.r.rule using oic.if.create",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrival on oic.if.create provides an empty payload.",
+            "x-example":
+            {}
+          }
+        }
+      },
+      "post": {
+        "description": "",
+        "parameters": [
+          {"$ref": "#/parameters/interface-create"},
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/rule-create" },
+            "x-example":
+              {
+                "rt": ["oic.r.rule.expression"],
+                "if": ["oic.if.rw","oic.if.baseline"],
+                "p":  {"bm": 3},
+                "rep": {
+                  "rule": "(timeperiod:triggered = true)",
+                  "ruleresult": false,
+                  "ruleenable": false,
+                  "actionenable": false
+                }
+              }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description" : "",
+            "x-example": {
+              "href": "/newURIforResource",
+              "ins": 39724818,
+              "rt": ["oic.r.rule.expression"],
+              "if": ["oic.if.rw","oic.if.baseline"],
+              "p":  {"bm": 3},
+              "rep": {
+                "rule": "(timeperiod:triggered = true)",
+                "ruleresult": false,
+                "ruleenable": false,
+                "actionenable": false
+              }
+            },
+            "schema": { "$ref": "#/definitions/rule-create" }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-all": {
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.ll","oic.if.baseline","oic.if.create"]
+    }
+  },
+  "definitions": {
+    "rule-create": {
+      "type": "object",
+      "properties": {
+        "href": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/href"
+        },
+        "if": {
+          "description": "The OCF Interfaces supported by the created Resource",
+          "items": {
+            "enum": [
```

Comment:
Only oic.if.baseline and oic.if.rw are allowed since they are the OCF Interfaces exposed by the Rule Expression Resource, right?

Reply:
Yes.  I'll fix this.